core/dbt/include/global_project/macros/relations/materialized_view/rename.sql

```diff
@@ -1,4 +1,12 @@
 {% macro get_rename_materialized_view_sql(relation, new_name) %}
+    {% set database = relation.database %}
+    {% set schema = relation.schema %}
+    {% set to_relation = adapter.get_relation(database=database, schema=schema, identifier=new_name) %}
+    {% if relation is not none %}
+        {% if to_relation is not none %}
+            {{ adapter.cache_renamed(from_relation=relation, to_relation=to_relation) }}
+        {% endif %}
+    {% endif %}
```

Comment:
@mikealfare Will this work or be a issue with the flow of 
`get_create_backup_sql` and `get_rename_intermediate_sql` both calling `get_rename_sql` which then calls `get_rename_materialzied_view_sql`

so we would technically be hitting our addition twice?

Reply:
This should go in the generic macro, I think it's called `get_rename_sql`.

Reply:
I've tried moving this up, into the generic default call, and initial dispatch call and `still get a cache inconsistency error for orders already existing`